core/modules/url_filter.h

```diff
@@ -21,6 +24,9 @@ class Flow {
   uint32_t dst_ip;
   uint16_t src_port;
   uint16_t dst_port;
+  uint32_t padding;
+
+  Flow() : padding(0) {}
```

Comment:
I might be wrong, but I think the default constructor for individual member fields with a primitive type is no-op... thus without an explicit class constructor the padding won't be zero-initialized by simply doing `Flow f;`.